TaffoInitializer/IndirectCallPatcher.cpp

```diff
@@ -0,0 +1,174 @@
+#include "IndirectCallPatcher.h"
+
+#include "Metadata.h"
+#include "TaffoInitializerPass.h"
+#include "TypeUtils.h"
+#include "llvm/ADT/Statistic.h"
+#include "llvm/IR/Constants.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/InstIterator.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/IR/Module.h"
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/Debug.h"
+#include <llvm/IR/IRBuilder.h>
+#include <llvm/Transforms/Utils/Cloning.h>
+#include <llvm/Transforms/Utils/ValueMapper.h>
```

Comment:
Can you uniform the include style?

Reply:
Specifically they should be all "" not <> (<> should be reserved for OS-level or STL headers)

Reply:
Standardized imports with "" and fixed order following `clang-format`